src/Npgsql.NodaTime/TimestampHandler.cs

```diff
@@ -10,11 +10,10 @@ namespace Npgsql.NodaTime
 {
     public class TimestampHandlerFactory : NpgsqlTypeHandlerFactory<Instant>
     {
-        // Check for the legacy floating point timestamps feature
         protected override NpgsqlTypeHandler<Instant> Create(NpgsqlConnection conn)
         {
             var csb = new NpgsqlConnectionStringBuilder(conn.ConnectionString);
-            return new TimestampHandler(conn.HasIntegerDateTimes, csb.ConvertInfinityDateTime);
+            return new TimestampHandler(csb.ConvertInfinityDateTime);
```

Comment:
This can be onlined:
```suggestion
            return new TimestampHandler(conn.Connector.ConvertInfinityDateTime);
```

Reply:
Are you sure? I thought the connector was internal? (This file is in one of the plugins)

Reply:
Indeed. Opened #2355 to track this.

Reply:
Note that there's no perf impact here, as this is plugin creation time (happens once on a physical connection open). I'm not sure it's worth doing anything here...